docs/source/restore/examples.rst

```diff
@@ -0,0 +1,191 @@
+   The easiest way to achieve that is to restore schema with :doc:`sctool restore <restore-schema>` (so that's what has already been done in this example).
+   You don't need to follow this step if you are sure that the destination cluster has the correct schema of restored tables and
+   that those tables are `truncated <https://docs.scylladb.com/stable/cql/ddl.html#truncate-statement>`_.
+
+#. Restore the content of the tables using :ref:`sctool restore <sctool-restore>`.
+
+   .. code-block:: none
+
+     sctool restore -c prod-cluster -L s3:test-bucket -T snapshot_tag --restore-tables -K test_keyspace
+     restore/31043443-8214-42d3-be98-ed9771c48dde
+
+#. Ensure that restoring the content of the tables has finished using :ref:`sctool progress <task-progress>`.
+
+   .. code-block:: none
+
+     sctool progress restore/31043443-8214-42d3-be98-ed9771c48dde -c prod-cluster --details
+     Run:            e10ec718-a7bf-11ed-b05a-0892040e83bb
+     Status:         DONE - repair required (see restore docs)
+     Start time:     16 Feb 23 17:17:29 CET
+     End time:       16 Feb 23 17:18:44 CET
+     Duration:       1m15s
+     Progress:       100% | 100%
+     Snapshot Tag:   sm_20230216160631UTC
+
+     ╭───────────────┬─────────────┬────────┬─────────┬────────────┬──────────────┬────────╮
+     │ Keyspace      │    Progress │   Size │ Success │ Downloaded │ Deduplicated │ Failed │
+     ├───────────────┼─────────────┼────────┼─────────┼────────────┼──────────────┼────────┤
+     │ test_keyspace │ 100% | 100% │ 7.938M │  7.938M │     7.938M │            0 │      0 │
+     ╰───────────────┴─────────────┴────────┴─────────┴────────────┴──────────────┴────────╯
+
+     Keyspace: test_keyspace
+     ╭────────┬─────────────┬────────┬─────────┬────────────┬──────────────┬────────┬────────────────────────┬────────────────────────╮
+     │ Table  │ Progress    │ Size   │ Success │ Downloaded │ Deduplicated │ Failed │ Started at             │ Completed at           │
+     ├────────┼─────────────┼────────┼─────────┼────────────┼──────────────┼────────┼────────────────────────┼────────────────────────┤
+     │ table1 │ 100% | 100% │ 2.642M │  2.642M │     2.642M │            0 │      0 │ 16 Feb 23 17:17:35 CET │ 16 Feb 23 17:18:37 CET │
+     ├────────┼─────────────┼────────┼─────────┼────────────┼──────────────┼────────┼────────────────────────┼────────────────────────┤
+     │ table2 │ 100% | 100% │ 2.657M │  2.657M │     2.657M │            0 │      0 │ 16 Feb 23 17:17:31 CET │ 16 Feb 23 17:18:33 CET │
+     ├────────┼─────────────┼────────┼─────────┼────────────┼──────────────┼────────┼────────────────────────┼────────────────────────┤
+     │ table3 │ 100% | 100% │ 2.640M │  2.640M │     2.640M │            0 │      0 │ 16 Feb 23 17:17:39 CET │ 16 Feb 23 17:18:41 CET │
+     ╰────────┴─────────────┴────────┴─────────┴────────────┴──────────────┴────────┴────────────────────────┴────────────────────────╯
```

Comment:
@Michal-Leszczynski is it still valid ?
I tested the restore of the schema and tables by using snapshot created by backup smoke test.
The output is not split per tables:
```
➜  scylla-manager git:(ml/restore_procedure_docs_description) ✗ ./sctool.dev progress restore/a4abeea5-baec-40fc-ad4b-3d571fc790e3 -c 1ad3a558-9a01-4dc1-b95f-1066be2aba2c
Run:		52af46f3-cd5d-11ed-a961-8c8caab68c8b
Status:		DONE - repair required (see restore docs)
Start time:	28 Mar 23 13:40:22 CEST
End time:	28 Mar 23 13:40:33 CEST
Duration:	11s
Progress:	100% | 100%
Snapshot Tag:	sm_20230328104613UTC

╭─────────────────┬─────────────┬────────┬─────────┬────────────┬────────╮
│ Keyspace        │    Progress │   Size │ Success │ Downloaded │ Failed │
├─────────────────┼─────────────┼────────┼─────────┼────────────┼────────┤
│ backuptest_data │ 100% | 100% │ 3.594M │  3.594M │     3.594M │      0 │
╰─────────────────┴─────────────┴────────┴─────────┴────────────┴────────╯

```